modules/oneVideoBidAdapter.md

```diff
@@ -298,7 +302,7 @@ var adUnits = [
                     id: 1,
                     page: 'https://verizonmedia.com',
                     referrer: 'https://verizonmedia.com'
-                  },            
+                  },
```

Comment:
please move openrtb params out of bidder params and into the ad unit in your examples

Reply:
@patmmccann I have updated the file accordingly

Reply:
Thanks!